services/QuillLMS/app/controllers/pages_controller.rb

```diff
@@ -38,8 +38,8 @@ def home_new
     @title = 'Quill.org | Interactive Writing and Grammar'
     @description = 'Quill provides free writing and grammar activities for middle and high school students.'
     # default numbers are current as of 06/03/24
-    @number_of_sentences = $redis.get(NUMBER_OF_SENTENCES) || 1972000000
-    @number_of_students = $redis.get(NUMBER_OF_STUDENTS) || 8900000
+    @number_of_sentences = $redis.get(NUMBER_OF_SENTENCES) || 2000000000
```

Comment:
These to numbers should be abstracted into constants, since they are duplicated below, in `#impact`